### PR TITLE
Plan Overhaul revert 

### PR DESCRIPTION
**Pull request by claudiucelfilip**
#### Proposed Changes

Addresses 779-gh-Automattic/martech
* Opens a branch to merge other Plan overhaul revert changes
* Disables the `plans/pro-plan` and `plans/starter-plan` flags for all environments

#### Testing Instructions

* Go to `/start/plans`
* You should see the Legacy plans (ie. Personal, Premium, Business, Ecommerce)
* Go to `/plans/[siteid]`
* You should see the Legacy plans (ie. Personal, Premium, Business, Ecommerce)

#### TODO
- [x] Confirm any `paid plans`, `Pro plan` or `Starter plan`  mentions are rolled back to the previous texts (ex. Business plan)
- [x] Update/roll back failing e2e tests
- [x] Remove legacy plan notice from the /plans page

### CODE REVIEW

**Comment by gmovr** [comment]
@chickee0311  Could you test this via the live link -- and map out any references to Pro/Starter?

**Comment by gmovr** [comment]
Merged in https://github.com/Automattic/wp-calypso/pull/64400

File: test/e2e/specs/plans/plans__existing-free-site-upgrade.ts [hunk 1]
```diff
@@ -19,7 +19,11 @@ import { apiDeleteSite } from '../shared';
 
 declare const browser: Browser;
 
-describe(
+/**
```
Comment (worldomonation) [inline test/e2e/specs/plans/plans__existing-free-site-upgrade.ts]: **suggestion**: @claudiucelfilip  another way to disable a test without having to disable waking rules are to simply remove the "@group" from the JSDoc header.

**Review by worldomonation — COMMENTED** [review]
Looks to be good from KitKat perspective, with one minor suggestion.

File: test/e2e/specs/plans/plans__existing-free-site-upgrade.ts (hunk 1 above)
Comment (claudiucelfilip) [reply test/e2e/specs/plans/plans__existing-free-site-upgrade.ts]: > **suggestion**: @claudiucelfilip another way to disable a test without having to disable waking rules are to simply remove the "@group" from the JSDoc header.

Thanks for this!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7450970

Hi @claudiucelfilip, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include this string: `With your plan, all WordPress.com advertising has been removed from your site. You can upgrade to a Business plan to also remove the WordPress.com footer credit.`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7450970).

**Pull request merged**